### PR TITLE
Fix RealmList.isValid Javadoc

### PR DESCRIPTION
**Pull request by cmelchior**
@realm/java

### CODE REVIEW

**Comment by beeender** [comment]
:+1:

File: realm/realm-library/src/main/java/io/realm/RealmList.java [hunk 1]
```diff
@@ -108,10 +108,10 @@ public RealmList(E... objects) {
     }
 
     /**
-     * Checks if {@link io.realm.RealmResults} is still valid to use i.e. the {@link io.realm.Realm} instance hasn't
-     * been closed.
+     * Checks if the {@link RealmList} is managed by Realm and contains valid data i.e. the {@link io.realm.Realm}
+     * instance hasn't been closed.
```
Comment (zaki50) [inline realm/realm-library/src/main/java/io/realm/RealmList.java]: 'the {@link io.realm.Realm} instance hasn't been closed.`

should be 

`the {@link io.realm.Realm} instance hasn't been closed and the container Realm object has not been deleted.`

?

**Comment by stk1m1** [comment]
Other than what @zaki50 says, :+1:

**Comment by nhachicha** [comment]
:+1:

**Comment by dalinaum** [comment]
:+1:

**Pull request merged**